<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSearch</class>
 <widget class="QDockWidget" name="AdvancedSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Search</string>
  </property>
  <widget class="QWidget" name="searchWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Context</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QComboBox" name="findMode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>Courant file</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Courant project</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All Project</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>System Files</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remplacer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SearchText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>421</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="filter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>421</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>*.*</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="caseSensitive">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Case Sensitive</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wholeWords">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Whole words</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="regularExpression">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>regular expression</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File pattern</string>
    </property>
   </widget>
   <widget class="QWidget" name="findDirWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QComboBox" name="findDir">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Search dir</string>
     </property>
    </widget>
    <widget class="QToolButton" name="getDir">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
